requirements.txt

```diff
@@ -0,0 +1,29 @@
+appdirs==1.4.3
```

Comment:
Is this file automatically generated? If not, I'm wondering if listing dependencies of packages that are already listed in here are necessary.